### PR TITLE
Create GitHub action to build the plugin and create a draft release

### PR DESCRIPTION
**Pull request by vrutz**
[sc-185521]
The GitHub action automatically parses the version from the `plugin.json` file at the given commit (defaults to master, but it can be given any commit hash).
It then builds the plugin and creates a draft pre-release (generating release notes is not supported by the action at this time).

After running the action, the publisher must go to the release, write the release notes and publish it (this will create the tag)

### CODE REVIEW

**Comment by qcastel** [comment]
can I see an example of the output of this github action? Like can I run it on your fork?

**Comment by thtrunck** [comment]
[sc-185521]

**Comment by vrutz** [comment]
> can I see an example of the output of this github action? Like can I run it on your fork?

@qcastel Of course. Not sure you have access to my fork, but I'll try to add you there: https://github.com/vrutz/dss-plugin-eks-clusters-ghaction/actions/runs/9204502580

**Comment by vrutz** [comment]
> can I see an example of the output of this github action? Like can I run it on your fork?

It also seems like we can use self-hosted runners. I'll double-check what the limitations are on the number of GitHub actions runs, but we could also set up Jenkins to run them for us.

File: .github/workflows/create-plugin-release.yml [hunk 1]
```diff
@@ -3,9 +3,6 @@ name: Build and create pre-release for the EKS cluster plugin
 on:
```
Comment (vrutz) [inline .github/workflows/create-plugin-release.yml:3]: Check if linter can be run before build (linked to PR https://github.com/dataiku/dss-plugin-eks-clusters/pull/72)

**Comment by vrutz** [comment]
I have another GitHub action to test, so I'll create a new branch on my fork for this action and a new PR

**Pull request closed without merging**